lib/querystream.js

```diff
@@ -4,6 +4,7 @@
  * Module dependencies.
  */
 
+var CircularJSON = require('circular-json');
```

Comment:
Why is CircularJSON imported here? It is never used in this file.

Reply:
Not sure actually, there's a few files with the addition that don't need it.. not sure why